.../include/sys/poll.h:1:2: warning: #warning redirecting incorrect #include <sys/poll.h> to <poll.h> [-Wcpp]
    1 | #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
      |  ^~~~~~~

---

--- a/plugins/common.h
+++ b/plugins/common.h
@@ -120,7 +120,11 @@
 #include <locale.h>
 
 #ifdef HAVE_SYS_POLL_H
-# include "sys/poll.h"
+#ifdef __GLIBC__
+# include <sys/poll.h>
+#else
+# include <poll.h>
+#endif /* musl */
 #endif
 
 /*
